ibm/mas_devops/roles/suite_app_install/vars/aibroker.yml

```diff
@@ -0,0 +1,7 @@
+---
+# Application specific variables for aibroker
+mas_app_fqn: aibrokerapps.apps.mas.ibm.com
```

Comment:
AIBroker isn't a MAS app so I don't think this should be included

Reply:
this one is resolved